hoodie-client/src/main/java/com/uber/hoodie/io/HoodieMergeHandle.java

```diff
@@ -136,7 +136,8 @@ private String init(String fileId, Iterator<HoodieRecord<T>> newRecordsItr) {
       // Load the new records in a map
       logger.info("MaxMemoryPerPartitionMerge => " + config.getMaxMemoryPerPartitionMerge());
       this.keyToNewRecords = new ExternalSpillableMap<>(config.getMaxMemoryPerPartitionMerge(),
-          Optional.empty(), new StringConverter(), new HoodieRecordConverter(schema, config.getPayloadClass()));
+          Optional.empty(), new StringConverter(),
```

Comment:
we are here again.. 
@bvaradar @n3nash what could cause this folding/formatting change, if we have checkstyle & already pre formatted code?

Reply:
These are wraps that happened column width 100. Checkstyle and code-style configuration under style/ folder have 120 as wrap point.
 @n3nash : If you have not already imported style/intellij-java-google-style.xml, Can you  use them.

Reply:
I already use them, not sure what happened here.

Reply:
ok, Digged a bit.  @n3nash:  Can you verify IntelliJ -> Preferences-> Code Style -> Java -> Wrapping and Braces  is set to 100 (instead of 120).

Found an issue in IntelliJ where if you have existing code-styles (other than default) which are active, importing a new code-style does not seem to change this value. I am not able to reproduce this consistently though. Deleting all non-default code-styles and/or activating the Default Code-Style and then importing our code-style does the trick. 

Let me know how it goes.

Balaji.V

Reply:
I have the following : 

![screen shot 2018-04-04 at 3 20 54 pm](https://user-images.githubusercontent.com/2722167/38337818-ce0145d0-381b-11e8-8f4c-3989f2920a98.png)

Changing this to 120 fixes it.